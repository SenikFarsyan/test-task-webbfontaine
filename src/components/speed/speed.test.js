import React from 'react';
import { shallow } from 'enzyme';

import Speed from './speed.component';

it('it expect to render Speed component', () => {
  expect(shallow(<Speed />)).toMatchSnapshot();
});
