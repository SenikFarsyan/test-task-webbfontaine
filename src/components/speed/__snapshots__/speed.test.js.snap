// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it expect to render Speed component 1`] = `ShallowWrapper {}`;
